### PR TITLE
Show full date on hover of fuzzy date [risk: low]

### LINKED ISSUE #591: Fuzzy dates should show full date on hover
For example, the "last changed" date in the workspaces list. Also, it would be nice to have last changed displayed in the workspace information section on the summary page.

### PR DESCRIPTION
**Pull request by katevoss**
Fixes #591

### CODE REVIEW

**Review by zarsky-broad — APPROVED** [review]

**Comment by katevoss** [comment]
To Do:
- [x] Display last changed in the workspace information section on the summary page.

File: src/pages/workspaces/workspace/Notebooks.js [hunk 1]
```diff
@@ -92,17 +92,21 @@ class NotebookCard extends Component {
...
           div({ style: { display: 'flex', justifyContent: 'space-between' } },
             [title, notebookMenu]),
           jupyterIcon,
           div({ style: { display: 'flex', alignItems: 'flex-end' } }, [
-            div({ style: { fontSize: '0.8rem', flexGrow: 1, marginRight: '0.5rem' } }, [
-              'Last edited:',
-              div({}, Utils.makePrettyDate(updated))
+            h(TooltipTrigger, { content: Utils.makeCompleteDate(updated) }, [
+              div({ style: { fontSize: '0.8rem', flexGrow: 1, marginRight: '0.5rem' } }, [
+                'Last edited:',
```
Comment (panentheos) [inline src/pages/workspaces/workspace/Notebooks.js:107]: TOL: If we were willing to have just the date string itself be hoverable, then we could abstract this pattern out into a little component.

**Review by panentheos — APPROVED** [review]
Looks good.

File: src/pages/workspaces/workspace/Notebooks.js (hunk 1 above)
Comment (katevoss) [reply src/pages/workspaces/workspace/Notebooks.js:107]: Good idea, I'll keep that in mind for future tooltips.

**Pull request merged**